tgui/packages/tgui/styles/interfaces/PreferencesMenu.scss

```diff
@@ -156,14 +157,14 @@ $department_map: (
         }
 
         &--Captain {
-          border: 3px solid rgba(200, 200, 0, 1);
-
-          &:first-child {
-            border-top: 3px solid rgba(200, 200, 0, 1);
+          &.head {
```

Comment:
I also changed JobsPage a bit relating to this. not sure if you want these changes